platform/ui/src/components/StudyListFilter/StudyListFilter.tsx

```diff
@@ -15,6 +15,7 @@ const StudyListFilter = ({
   isFiltering,
   numOfStudies,
   onUploadClick,
+  getConfigureDataSourceComponent,
```

Comment:
`getConfigureDataSourceComponent` sounds not right, did you mean `getConfigurableDataSourceComponent` or similar?

Reply:
I actually meant "get the component to configure the data source", so I named it `getConfigureDataSourceComponent`. Thus I am ok with the existing name, but if you want I can change it to your suggestion.

Reply:
how about `getDatasourceConfigureComponent` (I think we use datasource everywhere maybe i'm wrong)

Reply:
It looks like we use dataSource (camel cased) everywhere. How about `getDataSourceConfigurationComponent`?